Source/DafnyRuntime/DafnyRuntimePython/_dafny/__init__.py

```diff
@@ -116,11 +116,71 @@ def flatten(self):
             e = q.pop()
             if isinstance(e, list):
                 l += e
+            elif isinstance(e, _SeqSlice):
```

Comment:
How do you feel about merging that with the previous case?

Reply:
I'm fine either way -- I'll leave it up to you

Reply:
I prefer merging